packages/babel-code-frame/src/index.js

```diff
@@ -180,15 +180,17 @@ export default function(
   if (!deprecationWarningShown) {
     deprecationWarningShown = true;
 
-    const deprecationError = new Error(
```

Comment:
I'm ok with just not doing the warning anymore and making this a breaking change. We can just fix this automatically with https://github.com/babel/babel-upgrade

Reply:
We can do this in a separate PR.